Doc/library/xml.dom.minidom.rst

```diff
@@ -156,6 +156,9 @@ module documentation.  This section lists the differences between the API and
       The :meth:`writexml` method now preserves the attribute order specified
       by the user.
 
+   .. versionchanged:: 3.9
+      The `standalone` argument was added.
```

Comment:
For reStructuredText in cpython, code styling is usually applied with double backticks.
```suggestion
      The ``standalone`` argument was added.
```